Packs/CommunityCommonScripts/ReleaseNotes/1_4_1.md

```diff
@@ -0,0 +1,7 @@
+
+#### Scripts
+
+##### New: BulkCloseIncidents
+
+- New: Added a new script- BulkCloseIncidents that closes incidents in bulk.
```

Comment:
The release note description does not follow the approved templates. Please rephrase the note to start with one of the approved prefixes, such as 'Added a new script...'.